test/e2e/volume_provisioning.go

```diff
@@ -184,6 +186,70 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 			Expect(err).To(HaveOccurred())
 			framework.Logf(err.Error())
 		})
+
+		It("should test that deleting a claim before the volume is provisioned deletes the volume. [Volume]", func() {
+			// This case tests for the regressions of a bug fixed by PR #21268
+			// REGRESSION: Deleting the PVC before the PV is provisioned can result in the PV
+			// not being deleted.
+			// NOTE:  Polls until no PVs are detected, times out at 5 minutes.
+
+			const raceAttempts int = 100
+			var suffix string = ns[len(ns)-5:] // match trailing random string of generated namespace
+			var regressedPVs []v1.PersistentVolume
+			var regressCount int
+
+			By("Creating a StorageClass")
```

Comment:
Can the code to create and verify a SC be move to its own func and shared by the `It("should not provision a volume in an unmanaged GCE zone` above too?